mkt/files/models.py

```diff
@@ -231,7 +230,7 @@ def cleanup_file(sender, instance, **kw):
             return
         # FIXME: This needs to check if the file is public or not and choose
         # the appropriate storage backend.
-        if filename and storage.exists(filename):
+        if filename and public_storage.exists(filename):
```

Comment:
I think this should just check both public and private (and remove the FIXME comment)